docs/creating-new/create/cmake.rst

```diff
@@ -452,12 +449,15 @@ On Windows:
 
 .. code-block:: none
 
-  > cd hunter
+  > git clone https://github.com/ruslo/polly
+  > cd polly
+  [polly]> set PATH=%CD%\bin;%PATH%
 
-  [hunter]> where polly.py
-  C:\...\bin\polly.py
+  > cd hunter
+  [hunter]> where polly.bat
+  C:\...\bin\polly.bat
 
-  [hunter]> polly.py --help
+  [hunter]> polly.bat --help
```

Comment:
Actually it's not working for me because I have no Python in PATH :)

If file association is set correctly we can check only `polly.py --help`. By the way `jenkins.py` use old name `build.py`:
* https://github.com/ruslo/hunter/blob/0e0e1f87c7b8ef7ace8673cb9380590ce5ee5f5c/jenkins.py#L144-L156